scene/gui/tab_bar.cpp

```diff
@@ -471,40 +594,123 @@ void TabBar::_notification(int p_what) {
 	}
 }
 
-void TabBar::_draw_tab(Ref<StyleBox> &p_tab_style, Color &p_font_color, int p_index, float p_x) {
+void TabBar::_draw_tab(Ref<StyleBox> &p_tab_style, Color &p_font_color, int p_index, float p_tab_offset) {
 	RID ci = get_canvas_item();
 	bool rtl = is_layout_rtl();
 
 	Color font_outline_color = get_theme_color(SNAME("font_outline_color"));
 	int outline_size = get_theme_constant(SNAME("outline_size"));
 	int hseparation = get_theme_constant(SNAME("h_separation"));
```

Comment:
There is no corresponding `v_separation` constant. (Even though hseparation does not work like I intuitively expected it to...)

Reply:
`h_separation` separates elements in the tab object, it actually has no bearing on the TabBar as a whole, since the tabs are still horizontally aligned, the behavior for controlling that separation is actually in the style boxes (which normally just uses the borders and margins)